### PR TITLE
s2542, ta 12024: cap cadc-access-* to version 2.0

### PR DESCRIPTION
**Pull request by hjeeves**

### CODE REVIEW

File: cadc-registry-server/build.gradle [hunk 1]
```diff
@@ -24,7 +24,7 @@ dependencies {
 
     compile 'org.opencadc:cadc-util:[1.2,)'
     compile 'org.opencadc:cadc-log:[1.0,)'
-    compile 'org.opencadc:cadc-access-control-identity:[1.0,)'
+    compile 'org.opencadc:cadc-access-control-identity:[1.0,2.0)'
```
Comment (pdowler) [inline cadc-registry-server/build.gradle]: compile time dependency on cadc-access-control-identity is probably a mistake from when we moved this code from service.git/reg into a library... should be removed.

Comment (hjeeves) [reply cadc-registry-server/build.gradle]: This build needs the reference - AcIdentityManager is passed in as a parameter for MemoryJobPersistence. Inside that, it's used only in one place: (but this is in the uws library.) MIght be argued there is a way to instantiate the MemoryJobPersistence class so it creates it's own manager?

partial output from gradle build:

warning: [options] bootstrap class path not set in conjunction with -source 1.7
/Users/jeevesh/jcentre_repos/reg/cadc-registry-server/src/main/java/org/opencadc/reg/server/OAIJobManager.java:70: error: cannot find symbol
import ca.nrc.cadc.auth.ACIdentityManager;
                       ^
  symbol:   class ACIdentityManager
  location: package ca.nrc.cadc.auth
/Users/jeevesh/jcentre_repos/reg/cadc-registry-server/src/main/java/org/opencadc/reg/server/OAIJobManager.java:85: error: cannot find symbol
        MemoryJobPersistence jp = new MemoryJobPersistence(new RandomStringGenerator(16), new ACIdentityManager(), 60L);
                                                                                              ^
  symbol:   class ACIdentityManager
  location: class OAIJobManager
2 errors
1 warning
:compileJava FAILED

FAILURE: Build failed with an exception.

Comment (hjeeves) [reply cadc-registry-server/build.gradle]: It's in the 'OAIJobManager' class in cadc-registry-server, which my ide claims is never used.

Comment (pdowler) [reply cadc-registry-server/build.gradle]: OK - ignore for now

**Pull request merged**